bcc/module.go

```diff
@@ -231,6 +239,32 @@ func (bpf *Module) AttachKretprobe(fnName string, fd int) error {
 	return bpf.attachProbe(evName, BPF_PROBE_RETURN, fnName, fd)
 }
 
+// AttachTracepoint attaches a tracepoint fd to a function
+func (bpf *Module) AttachTracepoint(name string, fd int) error {
```

Comment:
Can this be documented that `name` is in the format `category:name`?

Reply:
Sure, added.